crates/manifest/src/schema/v2.rs

```diff
@@ -197,7 +197,7 @@ pub struct Component {
 
 /// Component dependencies
 #[derive(Clone, Debug, Default, Serialize, Deserialize)]
-#[serde(try_from = "Map<DependencyName, ComponentDependency>")]
```

Comment:
Is the matching `TryFrom` still used? Might as well delete it if not.

Reply:
I thought it was used in a test somewhere but apparently not. I have deleted it.